dns-controller/cmd/dns-controller/main.go

```diff
@@ -54,6 +57,9 @@ func main() {
 	watchIngress := true
 	flags.BoolVar(&watchIngress, "watch-ingress", watchIngress, "Configure hostnames found in ingress resources")
 
+	dnsServer := ""
+	flag.StringVar(&dnsServer, "dns-server", dnsServer, "DNS Server")
```

Comment:
It's neither nodeup nor protokube.
It's the cloudup's template function:
https://github.com/vmware/kops/pull/9/files#diff-76c7b34d006bb2e3f464748d6aa9cfe4R145